File: app/src/main/java/workshop1024/com/xproject/model/filter/Filter.java
package workshop1024.com.xproject.model.filter;

public class Filter {
    //过滤器id
    private String mFilterId;
    //过滤器名称
    private String mFilterName;

    public Filter(String filterId, String filterName) {
        mFilterId = filterId;
        mFilterName = filterName;
    }

    public String getFilterId() {
        return mFilterId;
    }

    public void setFilterId(String filterId) {
        mFilterId = filterId;
    }

    public String getFilterName() {
        return mFilterName;
    }

    public void setFilterName(String filterName) {
        mFilterName = filterName;
    }
}

File: main/src/main/java/workshop1024/com/xproject/main/filter/data/Filter.kt
package workshop1024.com.xproject.main.filter.data

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "filters")
data class Filter(//过滤器id
        @PrimaryKey @ColumnInfo(name = "filterId") var mFilterId: String, //过滤器名称
        @ColumnInfo(name = "filterName") var mFilterName: String)
